src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -5,26 +5,39 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+import java.util.Arrays;
+
 
 import seedu.addressbook.data.person.ReadOnlyPerson;
 
 /**
  * Finds and lists all persons in address book whose name contains any of the argument keywords.
- * Keyword matching is case sensitive.
+ * Keyword matching is NOT case sensitive.
  */
 public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-sensitive) and displays them as a list with index numbers.\n"
+            + "the specified keywords (NOT case-sensitive) and displays them as a list with index numbers.\n"
             + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
             + "Example: " + COMMAND_WORD + " alice bob charlie";
 
     private final Set<String> keywords;
 
-    public FindCommand(Set<String> keywords) {
-        this.keywords = keywords;
+    public FindCommand(Set<String> keywords) { this.keywords = keywords; }
+
+    /**
+     * Returns a copy of keywords, in small case, in this command.
+     */
+    public Set<String> ToSmallCaseKeywords(Set<String> keywords) {
+        //Converting set to array to iterate through
+        String[] keywordsArray = keywords.toArray(new String[keywords.size()]);
+        for (int i = 0; i < keywordsArray.length; i++) {
```

Comment:
Nicely commented code, making it very readable and clear. A tip: You can use enhanced iterators to iterate thru the set here!